src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../TodoContext';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    completed,
+    title,
+  } = todo;
+
+  const { showedTodo, setShowedTodo } = useContext(TodoContext);
+
+  const handleMouseEnter = (event: React.MouseEvent<HTMLTableRowElement>) => {
+    event.currentTarget.classList.add('has-background-info-light');
+  };
+
+  const handleMouseLeave = (event: React.MouseEvent<HTMLTableRowElement>) => {
+    event.currentTarget.classList.remove('has-background-info-light');
+  };
```

Comment:
Don't manipulate DOM elements directly, create a state and add the class conditionally to the element(use classnames library)